server_fn_macro/src/lib.rs

```diff
@@ -274,6 +274,10 @@ impl Parse for ServerFnName {
             .map(|encoding| match encoding.to_string().as_str() {
```

Comment:
For this block, it's probably worth matching on `encoding.to_string().to_lowercase()` so you don't have to handle different capitalization combinations.

Reply:
Yeah, that would simplify things a bit in terms of code length.